SYCL/Basic/accessor/accessor.cpp

```diff
@@ -200,6 +200,47 @@ int main() {
     }
   }
 
+  // Local and image accessors
+  {
```

Comment:
Images are not supported on FPGA device.

Please add compile time guard to avoid JITing/running on accelerator device.